macrosynergy/panel/basket_performance.py

```diff
@@ -222,31 +287,61 @@ def basket_performance(df: pd.DataFrame, contracts: List[str], ret: str = "XR_NS
     """
 
     assert isinstance(ret, str), "return category must be a <str>."
+    if cry is not None:
+        assert isinstance(cry, str), "carry category must be a <str>."
```

Comment:
This would raise an error if cry was a list of carries, such as real and nominal carries. Please allow list of carry names and then assert that the instance is str or list of strings,